### PR TITLE
Made Major semantic changes to HTML and CSS

### PR DESCRIPTION
**Pull request by SmithVyne**
We made major semantic changes and validated both HTML and CSS. This webpage is ready for production.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,198 @@
+<!DOCTYPE html>
```
Comment (majovanilla) [inline index.html:1]: - [x] Check your file against the validator. You have one error.
Screenshot from 2020-03-25 11-55-15

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+node_modules/
+stylelint.config.js
```
Comment (majovanilla) [inline .gitignore]: - [x] Don't include your styling in the gitignore. You want it to be in your repo so Stickler can check your linters.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+  color: #606060;
+}
+
+.video .desc h1 {
+  font-size: 26px;
+}
+
+.video .desc .details .buttons {
+  text-align: right;
+}
```
Comment (majovanilla) [inline style.css]: - [x] The previous TSE made an observation here and will like to clarify it. When you have classes, there is not necessary to create the "path" that leads to the element you want to style. In this code, are you saying that you want the elements with the vide, desc, details and buttons classes to have the text aligned to the right? or are you saying that you want the text inside the buttons to be aligned to the right?

If the answer is yes to the first question, you are doing it right. If your answer is no, then you just need the buttons class in there and delete the rest. That's what classes are for.

You would need to specify such path when using tags, for example, when you want to style the main > div > buttons.

Make sure you are actually styling what you want and if necessary, make the changes in all cases.

**Review by majovanilla — CHANGES_REQUESTED** [review]
Hey team!

## Good job on this project! 👍

There are still some changes 👇 you need to implement to approve this request 🔍

### Required Changes: ✍️

- Always remember to check your HTML using the [validator](https://validator.w3.org/). You have some errors and warnings. Please fix them.

- Always remember to activate your repo in Stickler. I cannot see your linters working!! You must include the stylelint file in your repo.

- Improve your Readme file. It should be descriptive of the project. Here's a great [template](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) I strongly recommend you to use.

- Please write meaningful pull request messages. Check this [example](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)

- **Do not** close the original Pull Request. When you make the changes in a pull request you need to submit the same link again. That way we can follow the history of your commits, previous reviews and changes...all until your PR is approved.

### Optional Changes:

If you want to make your project even better 🔥💥, I have a few suggestions you can implement.  

These are completely [Optional]: 🦔

- Improve the section below the main video. Add some padding to the left to make it look like the original page and separate the buttons of the right using some margin between them.

## Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@majo)
I would be happy to help! 👩‍💻 


Happy coding 🤓

--Maria Reyes

**Comment by men32z** [comment]
Hi!
Congratulations!
The code is clean and organized, you have made almost all the changes but still missing this one. 

"Always remember to activate your repo in Stickler. I cannot see your linters working!! You must include the stylelint file in your repo."

the file .stickler.yml is still missing, you [can  see here](https://github.com/microverseinc/linters-config/tree/master/css) how to configure stickler in your project, if you have questions you can reach me in slack. 
Happy Coding! 🤓

- Luis Preza

Status: Request changes.

**Review by men32z — CHANGES_REQUESTED** [review]
Hi!
Congratulations!
The code is clean and organized, you have made almost all the changes but still missing this one.

"Always remember to activate your repo in Stickler. I cannot see your linters working!! You must include the stylelint file in your repo."

the file .stickler.yml is still missing, you [can see here ](https://github.com/microverseinc/linters-config/tree/master/css)how to configure stickler in your project, if you have questions you can reach me in slack.
Happy Coding! 🤓

Luis Preza

Status: Request changes.

**Comment by mikenath223** [comment]
### Pull request currently under review by Michgolden Ukeje (TSE).
Kindly exercise patience 🐱

**Review by mikenath223 — APPROVED** [review]
### Nice Job Team 👏👏👏

It's nice that you have worked on aspects of your projects mentioned in previous reviews. 👍

STATUS: APPROVED ✅

Don't forget to fill the project completion form👍
Good luck with the next project.

HAPPY CODING👨‍💻

**Pull request merged**